offer-manager-domain/src/main/groovy/life/qbic/business/offers/create/CreateOfferOutput.groovy

```diff
@@ -40,4 +40,16 @@ interface CreateOfferOutput extends UseCaseFailure {
    * @since 1.0.0
    */
   void calculatedPrice(double netPrice, double taxes, double overheads, double totalPrice)
+
+  /**
+   * Transfers the calculated net price, taxes, overheads and total price
+   * to an implementing class
+   * @param netPrice The net price calculated for the requested services
+   * @param taxes The amount of taxes for the requested services
+   * @param overheads The amount of overheads for the requested services
+   * @param totalPrice The total price for the requested services, includes taxes and overheads
+   * @param totalDiscountAmount The total discount amount applied on the offer
+   * @since 1.1.0
+   */
+  void calculatedPrice(double netPrice, double taxes, double overheads, double totalPrice, double totalDiscountAmount)
```

Comment:
would it make sense to split this method into its sub parts? Should we deprecate the smaller method otherwise?
calculatedNetPrice
calculatedTaxes
calculatedOverheads
calculatedTotalPrice
calculatedDiscountAmount

Reply:
I deprecated the smaller methods